grails-app/i18n/messages.properties

```diff
@@ -3119,6 +3119,7 @@ react.default.italian.label=Italiano
 react.default.spanish.label=Español
 react.default.portuguese.label=Português
 react.default.acoli.label=Acoli
+react.default.haitian.label=Haitian
```

Comment:
Haitian isn't a language and I want these names to be in the form that would be recognized by the native speaker, so this one should be Kreyòl.

Reply:
@jmiranda I aligned it with what is returned by `Locale.getDisplayName()` here https://github.com/openboxes/openboxes/blob/develop/grails-app/views/common/_footer.gsp#LL28C9-L29C11. Otherwise we would get different labels on gsp and react footers. But if you want to turn it into `Kreyòl`, then we can change the `getDisplayName` in the `common/_footer.gsp` into `g:message`.